integration-tests/ui-integration-tests/pom.xml

```diff
@@ -239,6 +239,23 @@
                             </resources>
                         </configuration>
                     </execution>
+                    <!--<execution>-->
```

Comment:
These will be removed after reasearch on testing framework for UI tests are done